src/main/java/dev/openfeature/sdk/ImmutableContext.java

```diff
@@ -16,8 +17,8 @@
 @SuppressWarnings("PMD.BeanMembersShouldSerialize")
 public final class ImmutableContext implements EvaluationContext {
 
-    @Delegate
-    private final Structure structure;
+    @Delegate(excludes = DelegateExclusions.class)
```

Comment:
`merge` was coming both from this delegation and the superclass, which causes some odd compiler warnings. It's just coming via inheritance now.